test/evp_extra_test.c

```diff
@@ -1006,10 +994,13 @@ static int test_EVP_SM2(void)
 
     /* now check encryption/decryption */
     /*
-     * Skip this since SM2 public key encrytion is not moved into
-     * default provider yet
+     * SM2 public key encrytion is not moved into default provider yet,
```

Comment:
Will this fall back to legacy implementation?

Reply:
Yes, it should.